src/shogun/regression/LinearRidgeRegression.cpp

```diff
@@ -8,7 +8,6 @@
  */
 #include <shogun/lib/config.h>
 
-#ifdef HAVE_LAPACK
```

Comment:
ah so this guard was just outdated?

I think we should use linalg rather than eigen3 directly inside though

Reply:
Yes, just outdated. I will look into that to find the possibility of using pure linalg.

Reply:
leave it as it is for now (you can already change the rest)
Then you could create this method in linalg, and then send another PR

Reply:
Or, you could just add a loop for now.